### PR TITLE
feat: counter (Wed, Feb 14, 2024 03:24:35 PM)

### PR DESCRIPTION
**Pull request by LSTND**

### CODE REVIEW

File: src/widgets/Header/ui/Icons/Icons.module.scss [hunk 1]
```diff
@@ -10,3 +10,21 @@
...
+  height: 17px;
+  border: 2px solid #000000;
+  background: #ffffff;
+  border-radius: 50%;
+  color: #000000;
+  font-size: 12px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```
Comment (FrontKid) [inline src/widgets/Header/ui/Icons/Icons.module.scss:30]: Breaks on 4000

File: src/widgets/Header/ui/Icons/Icons.tsx [hunk 2]
```diff
@@ -7,11 +7,16 @@ import css from './Icons.module.scss';
...
   return (
     <div className={css.icons}>
       <Icon className={css.icon} to="/login" type="account" />
       <Icon className={css.icon} type="like" />
-      <Icon className={css.icon} type="cart" />
+      <div className={css.iconBtn}>
+        <Icon className={css.icon} type="cart" />
+        <span className={css.cartCount}>{cartCount}</span>
+      </div>
     </div>
```
Comment (FrontKid) [inline src/widgets/Header/ui/Icons/Icons.tsx:20]: need to think how to make it better with redux store

**Review by FrontKid — CHANGES_REQUESTED** [review]

File: src/widgets/Header/ui/Icons/Icons.tsx (hunk 2 above)
Comment (LSTND) [reply src/widgets/Header/ui/Icons/Icons.tsx:20]: i didnt do header

**Review by FrontKid — CHANGES_REQUESTED** [review]
image

image

**Pull request merged**